community/i3-gaps/build

```diff
@@ -0,0 +1,33 @@
+#!/bin/sh -e
+
+# export V=1 for verbose output when building
+[ "${V:-0}" -eq 1 ] > /dev/null 2>&1 && set -x || :
+
+export CHOST="${CHOST:-x86_64-pc-linux-musl}"
+
+PATCHES="${PATCHES:-}"
+
+PATCHES="
+i3-4.16-musl-GLOB_TILDE.patch
+${PATCHES}
+"
+
+if [ -n "${PATCHES}" ]
+then
+	for patch in ${PATCHES}
+	do
+		if [ -z "${patch}" ] || [ ! -e "${patch}" ]
+		then
+			continue
+		fi
+		patch -p1 < "${patch}"
+	done
+fi
+
+autoreconf -fvi
```

Comment:
Is `autoreconf` necessary?

Reply:
It's not strictly necessary but helps to make sure all of the auto-generated files can be built correctly and used without issues rather than relying on the (potentially old) `./configure` script that gets shipped with the tarball. Most distros run `autoreconf` on all projects that use autotools.

Reply:
KISS doesn't run `autoreconf` unless the package doesn't provide a generated `configure` script. If `i3` provides `configure`, remove it. :+1: